packages/solutions/module-tools/src/builder/feature/redirect.ts

```diff
@@ -207,6 +207,8 @@ export const redirect = {
         const staticPattern = [
           `import $$VAR from '$MATCH'`,
           `import $$VAR from "$MATCH"`,
+          `export {$$VAR} from '$MATCH'`,
+          `export {$$VAR} from "$MATCH"`,
```

Comment:
how about `import * as $$VAR from` and `export * from`?

Reply:
And there may be some other cases, such as `import type`

Reply:
👍 Can we add some test cases for `import * as` and `export * from`

Reply:
sure, i'll do it. i have add case about `export * from`